assets/js/common/ActivityLogDetailsModal/ActivityLogDetailModal.jsx

```diff
@@ -36,6 +102,71 @@ const activityTypesLabels = {
   [USER_DELETION]: 'User Deleted',
   [PROFILE_UPDATE]: 'Profile Updated',
   [CLUSTER_CHECKS_EXECUTION_REQUEST]: 'Checks Execution Requested',
+  [HOST_ADDED_TO_CLUSTER]: 'Host Added to Cluster',
+  [HEARTBEAT_FAILED]: 'Heartbeat Failed',
+  [HEARTBEAT_SUCCEEDED]: 'Heartbeat Succeeded',
+  [HOST_CHECKS_HEALTH_CHANGED]: 'Checks Health Changed',
```

Comment:
This could potentially be a string manipulating function, I guess? Turn the event names into a string is a bit like writing a custom serializer, for the most part.

Reply:
Yes, it could be something like
```js
const activityTypesLabels = _.fromPairs(
  ACTIVITY_TYPES.map((type) => [
    type,
    type.split('_').map(_.capitalize).join(' '),
  ])
);
```

However I'd like to get some more feedback on the rendered entries in order to figure out the transformation rule.

Additionally let's take into account that there is not yet a consistent pattern throughout all the activity types. But we can improve as we proceed.